protocol/0050-EPOC-epochs.md

```diff
@@ -93,7 +93,7 @@ The delegator cannot move the tokens before the epoch ends, they remain locked.
 The action can be announced at any time and is executed immediately following the block
 it is announced in. However, the stake is still counted as delegated to the validator until
 the last block of the epoch, though the delegator rewards are not paid to the delegator, but
-into an appropriate vega pool (the insurance pool, for example). The tokens are
+into an appropriate vega pool (the global insurance pool, for example [TODO: (WG) confirm]). The tokens are
```

Comment:
@ze97286 could you confirm what happens with these? I couldn't infer it from the code.

Reply:
none of this is correct. when one undelegateNow, their funds are immediately released, won't count for the reward, and hence won't be paid to anyone, surely not some [global] insurance pool. Regardless of implementation this doesn't make any sense. Suppose have my tokens delegated to some validator, on the first block of the epoch I undelegateNow all of it, why should staking reward consider this as if it was there fore the whole epoch and make some reward payout based on it (just not to the delegator)?

Reply:
Thanks, rephrased now. Does that mean we can delete the "Undelegate in anger" paragraph below it?

Reply:
as a command, yes. To do that one would just unstake vs undelegate.

Reply:
ok, removed now